.github/workflows/test-suite-native-e2e-ios.yml

```diff
@@ -0,0 +1,149 @@
+name: "[Test] suite-native iOS E2E"
+
+on:
+  schedule:
+    # Runs at midnight (00:00) on every Monday
+    # So far the iOS do not support Trezor device connection, so it is not necessary to run tests every day.
+    # Will be probably changed in the future.
+    - cron: "0 0 * * 1"
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "suite-native/**"
+      - "suite-common/**"
+      - "packages/connect/**"
+      - ".github/workflows/test-suite-native-e2e-android.yml"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  NODE_ENV: "test"
+  DETOX_BINARY_PATH: ../../node_modules/.bin/detox
+
+jobs:
+  prepare_ios_test_app:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v4
+        with:
+          submodules: "true"
+
+      - name: Install node and yarn
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+          cache: yarn
+
+      - name: Setup node_modules cache
+        uses: actions/cache@v4
+        with:
+          path: node_modules
+          key: node_modules/${{ github.ref }}/${{github.run_id}}
+
+      - name: Install Yarn dependencies
+        run: |
+          echo -e "\nenableScripts: false" >> .yarnrc.yml
+          echo -e "\nenableHardenedMode: false" >> .yarnrc.yml
+          yarn install
+
+      - name: Prebuild native expo project
+        working-directory: ./suite-native/app
+        run: yarn prebuild --platform ios --clean
+
+      - name: Sign message system config
+        working-directory: ./suite-common/message-system
+        run: yarn sign-config
+
+      - name: Use latest stable Xcode
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest-stable
+
+      - name: Build a Detox test app
+        working-directory: ./suite-native/app
+        run: $DETOX_BINARY_PATH build  --configuration ios.sim.release
+
+      - name: Save build to cache
```

Comment:
I'm not sure about saving the build to cache. I think it should be saved as an artifact and passed down to the test job. The cache has limited storage, and it's possible (though highly unlikely) that it will get purged before you get to use the build.